projects/admin/src/app/record/detail-view/address-type/address-type.component.html

```diff
@@ -0,0 +1,85 @@
+<!--
+  RERO ILS UI
+   Copyright (C) 2019 RERO
+  
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, version 3 of the License.
+  
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU Affero General Public License for more details.
+  
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<article>
+  <section class="m-2 p-2">
+    <!-- CONTACT_PERSON -->
+    <dl class="row mb-0" *ngIf="addressType.contact_person">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'Contact person' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        {{ addressType.contact_person }}
+      </dd>
+    </dl>
+    <!-- STREET -->
+    <dl class="row mb-0" *ngIf="addressType.street">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'Street' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        {{ addressType.street }}
+      </dd>
+    </dl>
+    <!-- POSTAL_CODE -->
+    <dl class="row mb-0" *ngIf="addressType.postal_code">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'Postal code' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        {{ addressType.postal_code }}
+      </dd>
+    </dl>
+    <!-- CITY -->
+    <dl class="row mb-0" *ngIf="addressType.city">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'City' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        {{ addressType.city }}
+      </dd>
+    </dl>
+    <!-- COUNTRY -->
+    <dl class="row mb-0" *ngIf="addressType.country">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'Country' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        {{ addressType.country }}
+      </dd>
+    </dl>
+  </section>
+  <section class="m-2 p-2">
```

Comment:
What is the logic behind these two sections?

Reply:
I replaced `section` with `div`